aws_lambda_powertools/metrics/metrics.py

```diff
@@ -71,15 +71,15 @@ def do_something():
         When metric object fails EMF schema validation
     """
 
-    _metrics = {}
-    _dimensions = {}
-    _metadata = {}
+    _metrics: Dict[str, Any] = {}
+    _dimensions: Dict[str, Any] = {}
```

Comment:
Did MyPy caused issues with `Dict[str, str]`?

Dimensions can only be strings.

```
>>> my_metrics._dimensions
{'test_dimension': 'test', 'test_dimension_2': 'test'}
```